footer done

johanngg7 opened the pull request:
Related to issue #137

johanngg7 committed:
footer done

miguel-merlin commented:
@SevenThanh Your changes did not pass prettier. Can you format your code using `npm run format:write`

johanngg7 committed:
formatted my code

johanngg7 commented:
- Deleted unnecessary parts
- Updated footer
- Mobile responsive
- Made it look pretty (font size, margin, etc)


<img width="496" alt="Screenshot 2024-07-21 at 3 51 39 PM" src="https://github.com/user-attachments/assets/3da64aec-4993-42a2-ba25-c10b369c3b0c">

<img width="1440" alt="Screenshot 2024-07-21 at 3 51 56 PM" src="https://github.com/user-attachments/assets/d920f374-66d8-4cbd-85d7-9920932d6986">

miguel-merlin commented on src/_components/Footer.jsx:
```diff
@@ -87,65 +84,39 @@ export default function Footer() {
-                <h2 className="mb-6 text-md font-semibold font- poppins text-white uppercase">
-                  Non-Profits
+                <h2 className="mb-6 text-md font-semibold font-poppins uppercase text-white">
+                  Students & Non-Profits
```
Could we change `General' and `Students & Non-Profits` to be online on mobile? I think it would look better if you had just two columns for each.

ShawnAviles replied:
I agree i think it would be nicer to have them be inline so that it would be a 2 column layout with `General` and `Students & Non-profits` on the right

miguel-merlin reviewed (commented):
Look amazing! I left a comment, otherwise great job!

ShawnAviles commented:
@miguel-merlin Should this be no longer merged as we have different footer designs to align with the newer target audience?

ShawnAviles commented:
Closing this PR as there are now newer designs for the footer that will be handled in a different issue

<img width="814" alt="Screenshot 2024-10-16 at 6 40 22 PM" src="https://github.com/user-attachments/assets/2fb3c83e-77fe-4efb-838e-8192b9a974fb">
